pkg/apis/swarm/v1alpha1/swarmdefaults.go.bak

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2018 Jeff Nickoloff (jeff@allingeek.com).
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package swarmdefaults
+
+import (
+	eccv1alpha1 "github.com/gotopple/kloudformation/pkg/apis/ecc/v1alpha1"
+	iamv1alpha1 "github.com/gotopple/kloudformation/pkg/apis/iam/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type SwarmDefaults struct {
```

Comment:
gotta stop doing this ".bak" thing. Use VCS.

Reply:
Removed. This is something I was messing around with